### PR TITLE
images are now stored in the static volume (update Runbook)

### PR DESCRIPTION
**Pull request by icyflame**
remove the steps in the deploy to prod section which requires the user to copy images before deleting and restore them afterwards.

### CODE REVIEW

File: RUNBOOK.md [hunk 1]
```diff
@@ -5,10 +5,8 @@
 0. Take a backup! Run `docker-compose exec backup run_backup.sh` and make sure it succeeded. Also run `git log` and note down the current deployed sha, in case you need to roll back.
 1. `git pull` and `docker-compose build`. This builds and caches the new images locally without interrupting the old server, which reduces downtime.
 2. `docker-compose down` shuts down the server and removes containers. Now downtime has started ticking.
-3. Copy any assets you need from the mediawiki volume `docker-compose cp ...`. Right now this is just images, but we should move everything to static so this step is unnecessary.
-4. `docker volume rm <mediawiki-volume>`, so that the mediawiki container can create a new volume with updates in the next step.
```
Comment (amrav) [inline RUNBOOK.md]: We still need this step unfortunately.

**Review by amrav — CHANGES_REQUESTED** [review]

File: RUNBOOK.md (hunk 1 above)
Comment (icyflame) [reply RUNBOOK.md]: yeah, actually, that step is marked as number 3 now! That's why the diff is weird. 🙂

**Comment by icyflame** [comment]
Used the alternate way to create markdown numbered lists which is slightly less intuitive, but works like unordered list (as in, markdown will figure out the number for each bullet point) 

Please see rendered diff for clarity.

**Review by amrav — APPROVED** [review]
Ah sorry, didn't see that! Don't mind either way.

**Pull request merged**